src/server/auth/server/api_server.go

```diff
@@ -139,7 +134,7 @@ func NewAuthServer(
 	public bool,
 	requireNoncriticalServers bool,
 	watchesEnabled bool,
-) (APIServer, error) {
+) (auth_server.APIServer, error) {
```

Comment:
I'm assuming we return an interface here because returning a private struct sucks. Could we just make the APIServer type public with all-private members? It's awkward to have `src/server/auth/server` reference `src/server/auth` like this.